src/main/java/com/bfrisco/benchants/utils/Toggle.java

```diff
@@ -0,0 +1,77 @@
+package com.bfrisco.benchants.utils;
+
+import org.bukkit.*;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.player.PlayerInteractEvent;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+import java.util.List;
+
+public class Toggle implements Listener {
+
+
+
+    @EventHandler
+    public static void activateClick(PlayerInteractEvent event) {
+
+        Player player = event.getPlayer();
+        Material coolDown = Material.JIGSAW;
+        if (!event.getAction().isRightClick()) return; //should reduce the amount of times the interactevent continues to use resources

```

Comment:
This comment can be removed - it's not only to be more performant, but it's also there for functionality (we don't want to apply ancient power on a left click, for example).